docs/src/guides/symfony/environment-variables.md

```diff
@@ -0,0 +1,302 @@
+
+  1. project wide route defined only by `{default}` or `{all}` (no path)
+  2. project wide route defined by `www.{default}` or `www.{all}` (no path)
+  3. route for the **current application** including `{default}` or `{all}` (might include a path)
+  4. route for the **current application** including `www.{default}` or `www.{all}` (might include a path)
+  5. first route for the current application
+  6. first route for the whole project
+
+  When several routes match a rule, the first one wins, the user order is kept. There's no preference regarding protocols.
+
+  {{< note title="Tip">}}
+
+  For multi-applications projects where several applications are publicly reachable, but one needs to determine the current application endpoint (for webhooks for example) and the project endpoint (to send emails for instance), an additional `SYMFONY_APPLICATION_DEFAULT_ROUTE_*` environment variables set is available. The same rules are applied to determine their value but only routes matching the current application are evaluated.
+
+  {{< /note >}}
+
+## Service Environment Variables
+
+
+When using the [Symfony integration](./integration), information about services
+are exposed via environment variables.
+
+List all exposed environment variables with the following command:
+
+```bash
+symfony ssh -- symfony var:export --multiline
+```
+
+Each exposed environment variable is prefixed by the relationship name. For
+instance, given the following relationships:
+
+```yaml
+relationships:
+    database: "securitydb:postgresql"
+```
+
+The environment variables for the database service will be prefixed by
+`DATABASE_`, the upper-cased version of the key defined in the relationship;
+like `DATABASE_URL`.
+
+While most environment variable names are derived from the relationship and
+service names, some are defined based on Symfony conventions like `MAILER_DSN`.
+
+{{< note theme="warning" >}}
+
+Be aware that service environment variables are not exposed when running
+the build hook script as services are not available during the build hook.
+
+{{< /note >}}
+
+The following sections lists all exposed environment variables that are automatically defined for each service:
+
+* [Emails](#emails)
+* [HTTP](#http)
+* [PostgreSQL](#postgresql)
+* [MySQL/MariaDB](#mysqlmariadb)
+* [MongoDB](#mongodb)
+* [Redis](#redis)
+* [RabbitMQ](#rabbitmq)
+* [Elasticsearch](#elasticsearch)
+* [Kafka](#kafka)
+* [InfluxDB](#influxdb)
+* [Memcached](#memcached)
+* [Solr](#solr)
+
+### Emails
+
+The configuration is exposed via the following environment variables:
+
+* `MAILER_ENABLED`: 1 if outgoing emails are enabled, 0 otherwise
+* `MAILER_DSN`/`MAILER_URL`: The Symfony-compatible mailer URL
+* `MAILER_HOST`: The SMTP server host
+* `MAILER_PORT`: The SMTP server port
+* `MAILER_TRANSPORT`: The SMTP transport mode (`smtp`)
+* `MAILER_AUTH_MODE`: The SMTP auth mode (`plain`)
+* `MAILER_USER`: The SMTP server user
+* `MAILER_PASSWORD`: The SMTP server password
```

Comment:
```suggestion
- `MAILER_ENABLED`: `1` if outgoing emails are enabled, `0` otherwise
- `MAILER_DSN`/`MAILER_URL`: The Symfony-compatible mailer URL
- `MAILER_HOST`: The SMTP server host
- `MAILER_PORT`: The SMTP server port
- `MAILER_TRANSPORT`: The SMTP transport mode (`smtp`)
- `MAILER_AUTH_MODE`: The SMTP auth mode (`plain`)
- `MAILER_USER`: The SMTP server user
- `MAILER_PASSWORD`: The SMTP server password
```